tests/summarywriter_test.cpp

```diff
@@ -53,6 +52,18 @@ TEST_CASE_METHOD(SummaryWriterTestsFixture, "Writing a scaler summary to file",
   }
 }
 
+/**
+ * Test text summary.
+ */
+TEST_CASE_METHOD(SummaryWriterTestsFixture, "Writing text summary to file",
+                 "[SummaryWriter]")
+{	
+  mlboard::SummaryWriter<mlboard::FileWriter>::Text("add Text support ", 1,
+      "mlpack is great", *f1);
```

Comment:
Let's not use add this as text.
```suggestion
      "Test case for checking text support in mlboard.", *f1);
```